deploy/ansible/ui/tasks/main.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Log into GHCR
+  community.docker.docker_login:
+    registry_url: ghcr.io
+    username: instructlab-bot
+    password: "{{ github_token }}"
+    reauthorize: true
+
+- name: Start the apiserver container
+  community.docker.docker_container:
+    name: apiserver
+    image: ghcr.io/instructlab/instructlab-bot/apiserver:main
+    state: started
+    pull: always
+    env:
+      LISTEN_ADDRESS: "${LISTEN_ADDRESS:-:3000}"
+      REDIS_SERVER: "${REDIS_SERVER:-redis:6379}"
+      API_USER: "${API_USER:-kitteh}"
+      API_PASS: "${API_PASS:-floofykittens}"
```

Comment:
Probably should not have defaults here right?

Reply:
Good catch. I just used from the podman compose. But i think for ansible I think it's better to set these in the secrets.enc. I will fix it.